### PR TITLE
Added missing securityContext block

### PR DESCRIPTION
**Pull request by teroahonen-vungle**
securityContext block was only applied to the coordinator, not to the worker.

### CODE REVIEW

File: valeriano-manassero/presto/README.md [hunk 1]
```diff
@@ -48,4 +48,4 @@ Deprecated, please use new Trino chart
 | tolerations | list | `[]` |  |
 
 ----------------------------------------------
-Autogenerated from chart metadata using [helm-docs v1.4.0](https://github.com/norwoodj/helm-docs/releases/v1.4.0)
+Autogenerated from chart metadata using [helm-docs v1.7.0](https://github.com/norwoodj/helm-docs/releases/v1.7.0)
```
Comment (valeriano-manassero) [inline valeriano-manassero/presto/README.md:51]: Pls don't change this file since it's not involved in PR and triggers checks for Presto that is deprecated.

File: valeriano-manassero/trino/Chart.yaml [hunk 2]
```diff
@@ -23,3 +23,5 @@ keywords:
   - sql
   - "query engine"
   - "high performance"
+  - "foobar"
```
Comment (valeriano-manassero) [inline valeriano-manassero/trino/Chart.yaml:26]: Can you pls remove foobar and foobar2 lines?

File: .github/workflows/release.yaml [hunk 3]
```diff
@@ -20,6 +20,6 @@ jobs:
       - name: Run chart-releaser
         uses: helm/chart-releaser-action@v1.0.0
         env:
-          CR_TOKEN: '${{ secrets.CR_TOKEN }}'
+          CR_TOKEN: '${{ secrets.GITHUB_TOKEN }}'
```
Comment (valeriano-manassero) [inline .github/workflows/release.yaml:23]: Why this change?

**Review by valeriano-manassero — CHANGES_REQUESTED** [review]
I added some comments that needs to be checked pls.

**Comment by teroahonen-vungle** [comment]
I ll create a new one, and close this...much easier....sorry.

**Comment by valeriano-manassero** [comment]
Don't need to be sorry, you are contributing 👍, thanks a lot!
